app-modules/alert/tests/Tenant/Filament/Actions/BulkCreateAlertActionTest.php

```diff
@@ -0,0 +1,120 @@
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use AdvisingApp\Alert\Models\Alert;
+use AdvisingApp\Alert\Tests\Tenant\Filament\Actions\RequestFactories\BulkCreateAlertActionRequestFactory;
+use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Pages\ListProspects;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages\ListStudents;
+use AdvisingApp\StudentDataModel\Models\Student;
+
+use function Pest\Laravel\assertDatabaseHas;
+use function Pest\Laravel\assertDatabaseMissing;
+use function Pest\Livewire\livewire;
+use function Tests\asSuperAdmin;
+
+it('shows the form and validation', function (BulkCreateAlertActionRequestFactory $data, array $errors) {
+    asSuperAdmin();
+
+    $student = Student::factory()->create();
+
+    $request = collect(BulkCreateAlertActionRequestFactory::new($data)->create());
+
+    livewire(ListStudents::class)
+        ->mountTableBulkAction('createAlert', [$student->getKey()])
+        ->setTableBulkActionData($request->toArray())
+        ->callMountedTableBulkAction()
+        ->assertHasTableBulkActionErrors($errors);
+
+    assertDatabaseMissing(Alert::class, $request->toArray());
+})->with([
+    'description required' => [
+        BulkCreateAlertActionRequestFactory::new()->without('description'),
+        ['description' => 'required'],
+    ],
+    'status_id required' => [
+        BulkCreateAlertActionRequestFactory::new()->without('status_id'),
+        ['status_id' => 'required'],
+    ],
+    'suggested_intervention required' => [
+        BulkCreateAlertActionRequestFactory::new()->without('suggested_intervention'),
+        ['suggested_intervention' => 'required'],
+    ],
+]);
+
+it('can successfully create bulk alert with student', function () {
+    asSuperAdmin();
+
+    $student = Student::factory()->create();
+
+    $request = collect(BulkCreateAlertActionRequestFactory::new()->create([
+        'concern_id' => $student->getKey(),
+        'concern_type' => $student->getMorphClass(),
```

Comment:
`concern_id` and `concern_type` are not fields in this form. A request factory should only be used to generate data for a form request.